lib/math.js

```diff
@@ -1,37 +1,8 @@
 /*
-* Shortens a number to the abbreviated thousands, millions, billions, etc
-* https://stackoverflow.com/a/40724354
-
-* @param {number} number: value to shorten
-* @returns {string|number}
-
-* @example
-* // return '12.3k'
-* abbreviateNumber(12345, 1)
-*/
-
-const SI_PREFIXES = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
-
-export const abbreviateNumber = (number, precision = 0) => {
-  // what tier? (determines SI prefix)
-  const tier = (Math.log10(number) / 3) | 0;
-
-  const round = value => {
-    return precision === 0 ? Math.round(value) : value.toFixed(precision);
-  };
-
-  // if zero, we don't need a prefix
-  if (tier === 0) {
-    return round(number);
-  }
-
-  // get prefix and determine scale
-  const scale = Math.pow(10, tier * 3);
-
-  // scale the number
-  const scaled = number / scale;
-
-  return scaled + SI_PREFIXES[tier];
+ * Adds commas to separate the digits of numbers (example: 100,000)
+ */
+export const formatCurrencyWithCommas = number => {
+  return number.toLocaleString('en-US');
```

Comment:
We should somehow consider the locale of the user here. In French for example, `,` is the separator for decimals (`42,50€` => `€42.50`).

Reply:
@Betree : Thanks, I've added some code to consider the browser locale so that if it's say french it's formatted that way. Here's a screenshot of how it looks. 🙂 

![image](https://user-images.githubusercontent.com/12435965/126934463-ea856173-6d04-48a3-96d2-363de6109866.png)

Let me know your thoughts.